python-clusters/create-aks-cluster/cluster.json

```diff
@@ -155,6 +155,36 @@
             "mandatory": false,
             "defaultValue": "Standard"
         },
+        {
+            "name": "upgradeChannel",
+            "label": "Cluster upgrades",
+            "description": "Determines whether and when the cluster is automatically upgraded based on new K8S or AKS releases",
+            "type": "SELECT",
+            "selectChoices": [
+                {"value":"none", "label":"No automatic upgrade"},
```

Comment:
Nitpicking here too, don't we want the order from the slowest to the fastest?
none, patch, stable, rapid, node-image?